src/cart/crt_register.c

```diff
@@ -406,20 +404,16 @@ crt_opc_reg(struct crt_opc_info *opc_info, crt_opcode_t opc, uint32_t flags,
 		opc_info->coi_input_size;
 
 	/* set RPC features */
-	disable_reply =  flags & CRT_RPC_FEAT_NO_REPLY;
-	enable_reset_timer = flags & CRT_RPC_FEAT_NO_TIMEOUT;
-
-	opc_info->coi_no_reply = disable_reply;
-	if (disable_reply)
-		D_DEBUG(DB_TRACE, "opc %#x, reply disabled.\n", opc);
-	else
-		D_DEBUG(DB_TRACE, "opc %#x, reply enabled.\n", opc);
-
-	opc_info->coi_reset_timer = enable_reset_timer;
-	if (enable_reset_timer)
-		D_DEBUG(DB_TRACE, "opc %#x, reset_timer enabled.\n", opc);
-	else
-		D_DEBUG(DB_TRACE, "opc %#x, reset_timer disabled.\n", opc);
+	opc_info->coi_no_reply = (flags & CRT_RPC_FEAT_NO_REPLY) ? 1 : 0;
+	opc_info->coi_reset_timer = (flags & CRT_RPC_FEAT_NO_TIMEOUT) ? 1 : 0;
+	opc_info->coi_queue_front = (flags & CRT_RPC_FEAT_QUEUE_FRONT) ? 1 : 0;
+
+	D_DEBUG(DB_TRACE, "opc %#x, reply %s.\n", opc,
```

Comment:
These lines of log are already fairly extensive at startup, would it be possible to combine the three lines into one?

Reply:
sure